include/cista/containers/string.h

```diff
@@ -74,16 +74,18 @@ struct generic_string {
 
   void set_owning(char const* str) { set_owning(str, mstrlen(str)); }
 
+  static constexpr msize_t len_limit = 15U;
```

Comment:
Name should be `short_string_length_limit` - otherwise one might think it also applies to heap allocated strings.

Reply:
short_length_limit would avoid duplicating the class name in the member's name.

Reply:
I wanted to make clear that it's about "small string optimization" or "short string optimization" (SSO). But `short_length_limit` also works.

Reply:
I would let short_length_limit